### ISSUE TITLE
Modernize Factor for .99

### ISSUE THREAD

**Issue by erg** [factor/factor#2039]

**Comment by catb0t** [comment]
under New Web Tech: #1899 HTML5 support / parsing (needs its own tracking issue)

**Comment by Immortalin** [comment]
WebAssembly support should be pretty straightforward

E.g.
https://github.com/remko/waforth

**Comment by Immortalin** [comment]
Running UI in web browser can also be naively solved if you compile factor to WebAssembly

**Comment by AlexIljin** [comment]
For the Windows/Linux UI notifications, I think you can link to issue #1951.

**Comment by phred** [comment]
We should probably update the Unicode tables so we can have all the latest codepoints and, for example :taco: emoji. Created #2335, will investigate.

**Comment by jonenst** [comment]
"Gtk2->gtk3" but gtk4 is out now

**Comment by erg** [comment]
Closing https://github.com/factor/factor/issues/1200 in favor of this ticket.

**Comment by zlandau** [comment]
For what it's worth, I've been (VERY slowly) working on a language server implementation. Who knows if I'll ever finish it, but I'm happy to bounce ideas off on people, if someone else is also looking into it.

**Comment by AndydeCleyre** [comment]
Are all of these blockers for the next release?

**Comment by alex-ilin** [comment]
@AndydeCleyre, no, it's just an abbreviated list of some of the secondary blockers.

**Comment by alex-ilin** [comment]
Actually, 0.99 will be released soon in the current state, and those things will get bumped to the next version.

**Comment by mrjbq7** [comment]
Unless anyone wants to fix or contribute anything!On Aug 8, 2023, at 10:04 AM, Alexander Ilin ***@***.***> wrote:﻿
Actually, 0.99 will be released soon in the current state, and those things will get bumped to the next version.
